File: Ejercicios de Semana Santa/ejercicios de semana santa java/Four java/Library/LicenciaC.java
package Library;

public class LicenciaC extends Licencia {
    public LicenciaC() {
        this.tipo = "C";
        this.costo = 1500000;
        this.duracion = 10;
    }

    public void mostrarDetalles() {
        System.out.println("\n--- Detalles de la Licencia ---");
        System.out.println("Tipo: " + tipo + " (Vehículos de servicio público)");
        System.out.println("Costo: " + String.format("%,.2f COP", costo));
        System.out.println("Duración: " + duracion + " meses");
    }
}

File: Prueba y error/ejercicios de semana santa php/Four php/Library/LicenciaC.php
<?php
require_once 'Licencia.php';

class LicenciaC extends Licencia {
    public function __construct() {
        $this->tipo = "Licencia C (Servicio público)";
        $this->duracion = 24;
        $this->costo = 350000;
    }
}